src/modules/ordersTable/utils/buildOrdersTableUrl.ts

```diff
@@ -0,0 +1,43 @@
+import { Path } from '@remix-run/router/history'
```

Comment:
Not from here, I know, but anyway:

> If you're using React Router, you should never import anything directly from the @remix-run/router

From https://www.npmjs.com/package/@remix-run/router

And since we don't have a direct dependency on it
![image](https://github.com/cowprotocol/cowswap/assets/43217/3afcdeb0-9fbf-4901-b103-63b520d7721a)

We shouldn't be importing from it